docs/proposals/002-api-proposal/proposal.md

```diff
@@ -80,101 +80,99 @@ The API design is based on these axioms:
  - Manage Pod lifecycle of pods within the pool 
 
-Additionally, any Pod that seeks to join a LLMServerPool would need to support a protocol, defined by LLM Instance Gateway, to ensure the Pool has adequate information to intelligently route requests.
+Additionally, any Pod that seeks to join an InferencePool would need to support a protocol, defined by this project, to ensure the Pool has adequate information to intelligently route requests.
 
-### LLMService
+### InferenceModel
 
-A LLMService allows the LLM Service Owner to define:
+An InferenceModel allows the Inference Workload Owner to define:
 - Which LoRA adapter(s) to consume 
-  - LLMService allows for traffic splitting between adapters _in the same LLMServerPool_ to allow for new LoRA adapter versions to be easily rolled out 
-- SLO objectives for the LLMService
-- The Pools this LLMService is relevant to 
+  - InferenceModel allows for traffic splitting between adapters _in the same InferencePool_ to allow for new LoRA adapter versions to be easily rolled out 
+- SLO objectives for the InferenceModel
+- The Pools this InferenceModel is relevant to 
 
 ### Spec
 
-**LLMService**
+**InferenceModel**
 ```golang
-// LLMService represents a set of LLM services that are multiplexed onto one 
-// or more backend pools. This resource is managed by the "LLM Service Owner"
-// persona. The Service Owner persona is: a team that trains, verifies, and
+// InferenceModel represents a set of Models/Adapters that are multiplexed onto one 
+// or more backend pools. This resource is managed by the "Inference Workload Owner"
+// persona. The Inference Workload Owner persona is: a team that trains, verifies, and
 // leverages a large language model from a model frontend, drives the lifecycle
 // and rollout of new versions of those models, and defines the specific
-// performance and latency goals for the model. These services are
-// expected to operate within a LLMServerPool sharing compute capacity with other
-// LLMServices, defined by the Inference Platform Admin. We allow a user who
-// has multiple LLMServices across multiple pools (with the same config) to
+// performance and latency goals for the model. These workloads are
+// expected to operate within an InferencePool sharing compute capacity with other
+// InferenceModels, defined by the Inference Platform Admin. We allow a user who
+// has multiple InferenceModels across multiple pools (with the same config) to
 // specify the configuration exactly once, and deploy to many pools 
 // simultaneously. Enabling a simpler config and single source of truth
-// for a given user. LLMService names are unique for a given LLMServerPool,
+// for a given user. InferenceModel names are unique for a given InferencePool,
 // if the name is reused, an error will be  shown on the status of a 
-// LLMService that attempted to reuse. The oldest LLMService, based on
+// InferenceModel that attempted to reuse. The oldest InferenceModel, based on
```

Comment:
This seems too much implementation detail of the validation logic. I think "InferenceModel names are unique for a given InferencePool" is enough.